content/en/logs/log_configuration/sensitive_data_detection.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Sensitive Data Detection
+kind: documentation
+aliases:
+    - /sensitive_data_detection/
```

Comment:
Did this page previously exist? If it's brand new, we can remove the alias as this is used solely for redirect.

Reply:
This is a new page. So I will remove the alias.